cmd/edgeql-go/testdata/test-project2/dbschema/migrations/00001.edgeql

```diff
@@ -0,0 +1,12 @@
+CREATE MIGRATION m1fqtauhtvc2w56wh2676x5g26aye22ghx7b7mtnbfekxpmxmnjx2a
```

Comment:
This migration file is not used? We don't have the `.esdl` files either.

Reply:
It is here to be sure that it is ignored by the code generator. If a `.go` file is generated and there is not matching `.go.assert` file then a test will fail.

Reply:
Ah makes a lot of sense! 👍